docs/en/metrics/metrics-alerting.asciidoc

```diff
@@ -19,4 +19,36 @@ The comparators available for conditions depend on the chosen field. The combina
 - Non-aggregatable fields: *matches*, *does not match*, *matches phrase*, *does not match phrase*. 
 
 [role="screenshot"]
-image::images/create-metrics-alert.png[Create metrics alert]
\ No newline at end of file
+image::images/create-metrics-alert.png[Create metrics alert]
+
+[float]
+==== Action messages
+
+You can insert the following variables into your action messages to provide information about an alert:
+
+[role="screenshot"]
+image::images/metrics-alert-message.png[Example alert message]
+
+**Basic variables**
+
+- **alertName**: The name of the alert
```

Comment:
Instead of highlighting each variable name in **bold**, I suggest using the code syntax. For example, `alertName`.